tutorials/sac-story-explore-2-get-familiar/sac-story-explore-2-get-familiar.md

```diff
@@ -0,0 +1,93 @@
+---
+author_name: Andrew Liu
+author_profile: https://github.com/andrewliu-SAP
+title: Get Familiar with the Story Interface
+description: Get familiar with SAP Analytics Cloud's story interface and learn how to interact with a story
+auto_validation: true
+time: 5
+tags: [ tutorial>beginner, topic>cloud]
+primary_tag: software-product>sap-analytics-cloud
+---
+
+## Prerequisites
+- You have access to an SAP Analytics Cloud account
+
+## Details
+### You will learn
+  - How to use the features in the toolbar
+  - How to navigate the pages of a story
+
+<!-- Add additional information: Background information, longer prerequisites -->
+---
+
+[ACCORDION-BEGIN [Step 1: ](Understand the Toolbar)]
+Let's first look at the toolbar. In the toolbar, you can save a copy, share with other users, add filters, navigate pages, toggle between View and Edit modes, and more.
+
+![Toolbar](Toolbar resized.png)
+
+
+|  Number         | Name                    | Description
+|  :------------- | :-------------          | :-------------
+|  1              | **File**                | Go here to export, share, schedule, and publish your story to the catalog.
+|  2              | **Edit**                | Reset your story to its original view state, refresh your story data, copy, and paste widgets.
+|  3              | **Tools**               | Edit Prompts for your data models, bookmark the current state of your story, run formulas on table cells, and  create Story Filters that filter data story-wide.
+|  4              | **Display**             | Toggle Comment Mode and Tab Display (page navigation option) on or off. We'll use these in Step 2.
+|  5              | **Quick Access Icons**  | Apply story filters, refresh data, and add bookmarks with just one click.
+|  6              | **Controls**            | View all filters applied to the page and widgets.
+|  7              | **Edit/View**           | Toggle between Edit and View modes. Edit mode enables a Design panel and a larger toolbar filled with features and capabilities that you can use to build stunning and interactive dashboards and reports.  Note that Edit mode is available only if you have rights to edit the story.
+
+<!-- 1. **File** - Go here to export, share, schedule, and publish your story to the catalog.
```

Comment:
Please remove sections that are not used.

Reply:
Done. Thanks